docs/docs/community/release-process.md

```diff
@@ -0,0 +1,75 @@
+---
+id: release-process
+title: The Nebari release process
+description: The Nebari release process and workflow
+---
+
+# The Nebari release process
+
+This page describes the high-level details of cutting a new Nebari release.
+
+|                   |                                                                                                                                                                                                  |
```

Comment:
Need to remove this row - otherwise it renders as an empty row

Reply:
I rotated the table instead 👍